server/views/partials/layout.njk

```diff
@@ -125,9 +123,11 @@
 {% block bodyEnd %}
   {# Run JavaScript at end of the
   <body>, to avoid blocking the initial render. #}
+  <script src="/assets/js/jquery.min.js"></script> 
```

Comment:
Moved `jquery` to the bottom for consistency with the guidance on https://design-patterns.service.justice.gov.uk/get-started/setting-up-javascript/